README.md

```diff
@@ -25,6 +25,8 @@ Crane (FinOps Crane) is a cloud native open source project which manages cloud r
     - [Analytics and Recommend Pod Resources](#analytics-and-recommend-pod-resources)
     - [Analytics and Recommend HPA](#analytics-and-recommend-hpa)
   - [RoadMap](#roadmap)
+  - [Contributing](#Contributing)
```

Comment:
use `contributing` instead of `Contributing` to keep same style with other items?